src/java.desktop/share/classes/java/awt/Button.java

```diff
@@ -122,21 +122,6 @@ public class Button extends Component implements Accessible {
     @Serial
     private static final long serialVersionUID = -8774683716313001058L;
 
-
-    static {
-        /* ensure that the necessary native libraries are loaded */
-        Toolkit.loadLibraries();
```

Comment:
I think it is safer to preserve loading the toolkit libraries: `Button` is a peered component and I can't see any of the peers loads the libraries.

Reply:
Button extends Component, which loads libraries

Reply:
another thing to note is that the peers are created by the toolkit which init awt lib on startup.